modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -10,6 +10,24 @@ Remember to xref:backup-restore/planning.adoc[plan your backup] carefully and ba
 Note that it is not allowed to take a backup of a database alias, only physical databases can be backed up.
 ====
 
+Before starting a backup, pay attention to your checkpointing policy, as the checkpoint has to be completed before the backup starts.
+
+Consider the following statements:
+
+* The backup cannot begin until the checkpoint finishes.
+
+* If the checkpoint takes longer than the configured backup timeout, this may cause the backup to fail.
+The configuration setting xref:configuration/configuration-settings.adoc#config_dbms.cluster.catchup.client_inactivity_timeout[`dbms.cluster.catchup.client_inactivity_timeout`] governs the backup timeout.
```

Comment:
This is not really a backup timeout. It just puts a restriction on how long the network can be idle (i.e. not receive any data). As long as the backup client is receiving data there's no real timeout on backups. So in this context, the backup job fails because it doesn't receive anything for as long as the server is checkpointing.